packages/harmony/Custom.js

```diff
@@ -0,0 +1 @@
+export const Custom = ({ children }) => <div data-testid='supp'>{children}</div>
```

Comment:
Mind adding a little documentation or a more descriptive name?

Reply:
sholud be removed lol